File: python/integracion_con_bases_de_datos/mini_sistema_completo/conexion.py
import mysql.connector
def conectar():
    try:
        conexion = mysql.connector.connect(
            host="localhost",
            user="root",
            password="",
            database="mini_sistema_completo"
        )
        return conexion
    except mysql.connector.Error as e:
        print("Error al conectar:", e)
        return None

File: visual_basic/integracion_con_base_de_datos/eliminar_contacto/conexion.vb
Imports MySql.Data.MySqlClient
Module conexion
    Public Function ObtenerConexion() As MySqlConnection
        Dim cadenaConexion As String = "server=localhost;user id=root;password=;database=eliminar_ese_contacto"
        Return New MySqlConnection(cadenaConexion)
    End Function
End Module